docs/easing.md

```diff
@@ -359,7 +359,7 @@ A useful tool to visualize cubic bezier curves can be found at http://cubic-bezi
 ### `in()`
 
 ```jsx
-static in(easing);
+static in easing;
```

Comment:
This is still a Prettier issue, can we [ignore](https://prettier.io/docs/en/ignore.html#markdown) this line? 😃

Reply:
Just this one line in this one file? Used these: https://prettier.io/docs/en/ignore.html

Anything else we should ignore?

Reply:
It seems like the only one place.